apps/src/templates/rubrics/RubricContent.jsx

```diff
@@ -163,6 +163,14 @@ export default function RubricContent({
           </div>
         </div>
       )}
+      {!studentLevelInfo && rubricLevel?.position && (
+        <Heading5>
```

Comment:
This might be out of scope for now, but curious if there's any way we could combine this with the same thing above 🤔

Reply:
Ah... yes... this component was/is tricky for me to parse.  I was able to move some `divs` around to be able to use the title inside the initial div.  It messed with some spacing but I have another ticket focused on narrowing ALL spacing so will clean that up in the next ticket.

Reply:
Yeah there's a lot going on, unfortunately. I figured the spacing might get messed up but as long as that's on your radar, that sounds good to me!